test/runner/lib/core_ci.py

```diff
@@ -106,6 +106,7 @@ def __init__(self, args, platform, version, stage='prod', persist=True, name=Non
 
                 display.info('Loaded existing %s/%s instance %s.' % (self.platform, self.version, self.instance_id),
                              verbosity=1)
+                display.info('Instance %s/%s on %s' % (self.platform, self.version, self._uri), verbosity=1)
```

Comment:
Since the previous log entry is at the same verbosity, repeating the platform and version a second time isn't needed. I'd just add the uri to the existing log entry:

```python
display.info('Loaded existing %s/%s instance from: %s' % (self.platform, self.version, self._uri),
```

Since the instance ID is in the uri, no need to show it twice.